bin/pre_runner_boot.rb

```diff
@@ -8,6 +8,6 @@
 
 Rails.configuration.cache_classes = true
 
-Dotenv.load if ENV['APP_SECRET_TOKEN'].blank?
+Dotenv.load if ENV['APP_SECRET_TOKEN'].blank? or ENV['APP_ENCRYPTION_PASSPHRASE'].blank?
```

Comment:
This might break stuff for people who don't have `APP_ENCRYPTION_PASSPHRASE` set, so I wouldn't add this.